Jenkins

JordanP opened the pull request.

Commit:
CI: Import `tox` job scripts from `ScalitySproxydSwift`

As of https://github.com/scality/ScalitySproxydSwift/commit/ba64ec984ad6

Commit:
Add functional tests

Commit:
Fix pep8

Commit:
Add functional tests: 1st try

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/6/
Test PASSed.

JordanP commented:
Retest this please.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-functional-tests/9/
Test FAILed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/7/
Test PASSed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-functional-tests/12/
Test FAILed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/8/
Test PASSed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-functional-tests/14/
Test FAILed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/9/<h2>Build result: FAILURE</span></h2>[...truncated 5 lines...] > git config remote.origin.url https://github.com/scality/scality-glance-store.git # timeout=10Fetching upstream changes from https://github.com/scality/scality-glance-store.git > git --version # timeout=10 > git -c core.askpass=true fetch --tags --progress https://github.com/scality/scality-glance-store.git +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/* > git rev-parse refs/remotes/origin/pr/8/merge^{commit} # timeout=10 > git rev-parse refs/remotes/origin/origin/pr/8/merge^{commit} # timeout=10Checking out Revision 2d70028818a571895d9eea59877dfda25b5aea03 (refs/remotes/origin/pr/8/merge) > git config core.sparsecheckout # timeout=10 > git checkout -f 2d70028818a571895d9eea59877dfda25b5aea03First time build. Skipping changelog.Triggering scality-glance-store-tox » py26,centos6-largeTriggering scality-glance-store-tox » py27,centos7-largeTriggering scality-glance-store-tox » py27,ubuntu14-largeTriggering scality-glance-store-tox » py27,ubuntu12-largeTriggering scality-glance-store-tox » pep8,ubuntu14-largeConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: Waiting for next available executor on centos6-largeConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: Waiting for next available executor on centos6-large-a3aConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: Waiting for next available executor on centos6-largescality-glance-store-tox » py26,centos6-large completed with result SUCCESSscality-glance-store-tox » py27,centos7-large completed with result SUCCESSscality-glance-store-tox » py27,ubuntu14-large completed with result SUCCESSscality-glance-store-tox » py27,ubuntu12-large completed with result FAILUREscality-glance-store-tox » pep8,ubuntu14-large completed with result SUCCESSSetting status of 372a8a95888747f9552377aead109cc7fe5f7346 to FAILURE with url https://37.187.159.67:5443/job/scality-glance-store-tox/9/ and message: Merged build finished.
Test FAILed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-functional-tests/16/
Test FAILed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/10/
Test PASSed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-functional-tests/17/
Test FAILed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/11/
Test PASSed.

JordanP commented:
Test passed here : https://37.187.159.67:5443/job/scality-glance-store-functional-tests/18/ Had to trigger the build manually though... 
The build triggered by github notification https://37.187.159.67:5443/job/scality-glance-store-functional-tests/17/ failed for some git-reason.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-functional-tests/19/
Test FAILed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/12/
Test PASSed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-functional-tests/21/
Test FAILed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/13/
Test PASSed.

JordanP commented:
Functional test passed : https://37.187.159.67:5443/job/scality-glance-store-functional-tests/host=ubuntu14-large/22/

NicolasT commented on jenkins/scality-glance-store-functional-tests/10-install-ring.sh:
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -xue
+
+git submodule init
```
This should be done by Jenkins already. If not, it should be configured to do so. Same for the `update`.

Commit:
Let Jenkins do the submodule init

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-functional-tests/23/
Test FAILed.

NicolasT commented on jenkins/scality-glance-store-tox/20-run-tox.sh:
```diff
@@ -0,0 +1,20 @@
+EOF
+fi
+
+tox
```
I believe the above is a very old version of that file. It doesn't report pep8 failures properly.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/14/<h2>Build result: FAILURE</span></h2>[...truncated 5 lines...] > git config remote.origin.url https://github.com/scality/scality-glance-store.git # timeout=10Fetching upstream changes from https://github.com/scality/scality-glance-store.git > git --version # timeout=10 > git -c core.askpass=true fetch --tags --progress https://github.com/scality/scality-glance-store.git +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/* > git rev-parse refs/remotes/origin/pr/8/merge^{commit} # timeout=10 > git rev-parse refs/remotes/origin/origin/pr/8/merge^{commit} # timeout=10Checking out Revision 2d8f6d8feb24b68173e5a9c029479ebd374bb130 (refs/remotes/origin/pr/8/merge) > git config core.sparsecheckout # timeout=10 > git checkout -f 2d8f6d8feb24b68173e5a9c029479ebd374bb130First time build. Skipping changelog.Triggering scality-glance-store-tox » py26,centos6-largeTriggering scality-glance-store-tox » py27,centos7-largeTriggering scality-glance-store-tox » py27,ubuntu14-largeTriggering scality-glance-store-tox » py27,ubuntu12-largeTriggering scality-glance-store-tox » pep8,ubuntu14-largeConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: All nodes of label ‘centos6-large’ are offlineConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: centos6-large-9e6 is offlineConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: Waiting for next available executor on centos6-large-9e6scality-glance-store-tox » py26,centos6-large completed with result SUCCESSscality-glance-store-tox » py27,centos7-large completed with result SUCCESSscality-glance-store-tox » py27,ubuntu14-large completed with result SUCCESSscality-glance-store-tox » py27,ubuntu12-large completed with result FAILUREscality-glance-store-tox » pep8,ubuntu14-large completed with result SUCCESSSetting status of 37199cafd33a5d8dd5b23a9b7f14cf74adb6f08a to FAILURE with url https://37.187.159.67:5443/job/scality-glance-store-tox/14/ and message: Merged build finished.
Test FAILed.

Commit:
Update 20-run-tox.sh to the latest version

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/15/
Test PASSed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-functional-tests/24/
Test PASSed.

JordanP commented:
\o/ double passed !

NicolasT commented on jenkins/scality-glance-store-functional-tests/extras.d/55-scality-glance-store.sh:
```diff
@@ -0,0 +1,28 @@
+# 55-scality-glance-store - Devstack extra script to configure Glance API with Scality store
+
+function install_scality_store {
+    sudo pip install https://github.com/scality/scality-sproxyd-client/archive/master.tar.gz
```
Could this be parametrized, so we can have the CI job work on several branches/releases/...?

NicolasT commented on jenkins/scality-glance-store-functional-tests/extras.d/55-scality-glance-store.sh:
```diff
@@ -0,0 +1,28 @@
+    # complaining about an old version of python-six.
+    # Probably a setuptools version issue
+    # sudo python setup.py install
+    sudo pip install https://github.com/scality/scality-glance-store/archive/master.tar.gz
```
Could this be parametrized, so we can have the CI job work on several branches/releases/...?

JordanP replied:
So far we only have and test master. I don't see use maintaining several versions of this in the neat future. When needed, I agree, this would have to be changed.

NicolasT commented on jenkins/scality-glance-store-tox/10-install-deps.sh:
```diff
@@ -0,0 +1,37 @@
+}
+
+function install_centos {
+    sudo yum -y install python-devel libffi-devel;
```
Guess all these calls to `yum` could be turned into one.

JordanP replied:
Will do.

Commit:
Merge several yum install calls

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-functional-tests/26/<h2>Build result: FAILURE</span></h2>GitHub pull request #8 of commit d299ad0b6234b2b0b3c15ddeac35a4f2a0fe650b automatically merged.Setting status of d299ad0b6234b2b0b3c15ddeac35a4f2a0fe650b to PENDING with url https://37.187.159.67:5443/job/scality-glance-store-functional-tests/26/ and message: Merged build started.Building on master in workspace /var/lib/jenkins/workspace/scality-glance-store-functional-tests > git rev-parse --is-inside-work-tree # timeout=10Fetching changes from the remote Git repository > git config remote.origin.url https://github.com/scality/scality-glance-store.git # timeout=10Fetching upstream changes from https://github.com/scality/scality-glance-store.git > git --version # timeout=10 > git -c core.askpass=true fetch --tags --progress https://github.com/scality/scality-glance-store.git +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/* > git rev-parse refs/remotes/origin/pr/8/merge^{commit} # timeout=10 > git rev-parse refs/remotes/origin/origin/pr/8/merge^{commit} # timeout=10Checking out Revision cd60e534cc8c3bea9faa7d94605ab9b6fc946701 (refs/remotes/origin/pr/8/merge) > git config core.sparsecheckout # timeout=10 > git checkout -f cd60e534cc8c3bea9faa7d94605ab9b6fc946701 > git rev-list 20c5d4052ba89b7a6d1429c28b95edf48eecb5b8 # timeout=10 > git remote # timeout=10 > git submodule init # timeout=10 > git submodule sync # timeout=10 > git config --get remote.origin.url # timeout=10 > git submodule update --init --recursiveTriggering scality-glance-store-functional-tests » ubuntu14-largeConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: All nodes of label ‘ubuntu14-large’ are offlineConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: ubuntu14-large-6e1 is offlinescality-glance-store-functional-tests » ubuntu14-large completed with result FAILURESetting status of d299ad0b6234b2b0b3c15ddeac35a4f2a0fe650b to FAILURE with url https://37.187.159.67:5443/job/scality-glance-store-functional-tests/26/ and message: Merged build finished.
Test FAILed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/17/<h2>Build result: FAILURE</span></h2>[...truncated 5 lines...] > git config remote.origin.url https://github.com/scality/scality-glance-store.git # timeout=10Fetching upstream changes from https://github.com/scality/scality-glance-store.git > git --version # timeout=10 > git -c core.askpass=true fetch --tags --progress https://github.com/scality/scality-glance-store.git +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/* > git rev-parse refs/remotes/origin/pr/8/merge^{commit} # timeout=10 > git rev-parse refs/remotes/origin/origin/pr/8/merge^{commit} # timeout=10Checking out Revision cd60e534cc8c3bea9faa7d94605ab9b6fc946701 (refs/remotes/origin/pr/8/merge) > git config core.sparsecheckout # timeout=10 > git checkout -f cd60e534cc8c3bea9faa7d94605ab9b6fc946701First time build. Skipping changelog.Triggering scality-glance-store-tox » py26,centos6-largeTriggering scality-glance-store-tox » py27,centos7-largeTriggering scality-glance-store-tox » py27,ubuntu14-largeTriggering scality-glance-store-tox » py27,ubuntu12-largeTriggering scality-glance-store-tox » pep8,ubuntu14-largeConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: All nodes of label ‘centos6-large’ are offlineConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: centos6-large-e62 is offlineConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: Waiting for next available executor on centos6-large-e62scality-glance-store-tox » py26,centos6-large completed with result FAILUREscality-glance-store-tox » py27,centos7-large completed with result FAILUREscality-glance-store-tox » py27,ubuntu14-large completed with result FAILUREscality-glance-store-tox » py27,ubuntu12-large completed with result FAILUREscality-glance-store-tox » pep8,ubuntu14-large completed with result UNSTABLESetting status of d299ad0b6234b2b0b3c15ddeac35a4f2a0fe650b to FAILURE with url https://37.187.159.67:5443/job/scality-glance-store-tox/17/ and message: Merged build finished.
Test FAILed.

JordanP commented:
jenkins, test this please.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/18/<h2>Build result: FAILURE</span></h2>[...truncated 11 lines...] > git config remote.origin.url https://github.com/scality/scality-glance-store.git # timeout=10Fetching upstream changes from https://github.com/scality/scality-glance-store.git > git -c core.askpass=true fetch --tags --progress https://github.com/scality/scality-glance-store.git +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/* > git rev-parse refs/remotes/origin/pr/8/merge^{commit} # timeout=10 > git rev-parse refs/remotes/origin/origin/pr/8/merge^{commit} # timeout=10Checking out Revision cd60e534cc8c3bea9faa7d94605ab9b6fc946701 (refs/remotes/origin/pr/8/merge) > git config core.sparsecheckout # timeout=10 > git checkout -f cd60e534cc8c3bea9faa7d94605ab9b6fc946701First time build. Skipping changelog.Triggering scality-glance-store-tox » py26,centos6-largeTriggering scality-glance-store-tox » py27,centos7-largeTriggering scality-glance-store-tox » py27,ubuntu14-largeTriggering scality-glance-store-tox » py27,ubuntu12-largeTriggering scality-glance-store-tox » pep8,ubuntu14-largeConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: Waiting for next available executorConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: All nodes of label ‘centos6-large’ are offlineConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: centos6-large-cb is offlineConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: Waiting for next available executor on centos6-large-cbscality-glance-store-tox » py26,centos6-large completed with result FAILUREscality-glance-store-tox » py27,centos7-large completed with result FAILUREscality-glance-store-tox » py27,ubuntu14-large completed with result FAILUREscality-glance-store-tox » py27,ubuntu12-large completed with result FAILUREscality-glance-store-tox » pep8,ubuntu14-large completed with result UNSTABLESetting status of d299ad0b6234b2b0b3c15ddeac35a4f2a0fe650b to FAILURE with url https://37.187.159.67:5443/job/scality-glance-store-tox/18/ and message: Merged build finished.
Test FAILed.

Commit:
Bound Tox version to <=1.9.0

Tox 1.9.1 doesn"t seem to follow git+https://github.com/scality/scality-sproxyd-client.git#egg=scality-sproxyd-client
properly

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/19/<h2>Build result: FAILURE</span></h2>[...truncated 5 lines...] > git config remote.origin.url https://github.com/scality/scality-glance-store.git # timeout=10Fetching upstream changes from https://github.com/scality/scality-glance-store.git > git --version # timeout=10 > git -c core.askpass=true fetch --tags --progress https://github.com/scality/scality-glance-store.git +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/* > git rev-parse refs/remotes/origin/pr/8/merge^{commit} # timeout=10 > git rev-parse refs/remotes/origin/origin/pr/8/merge^{commit} # timeout=10Checking out Revision ca4b1fec2bda5c9f1069be7a11a9553f354c3f0b (refs/remotes/origin/pr/8/merge) > git config core.sparsecheckout # timeout=10 > git checkout -f ca4b1fec2bda5c9f1069be7a11a9553f354c3f0bFirst time build. Skipping changelog.Triggering scality-glance-store-tox » py26,centos6-largeTriggering scality-glance-store-tox » py27,centos7-largeTriggering scality-glance-store-tox » py27,ubuntu14-largeTriggering scality-glance-store-tox » py27,ubuntu12-largeTriggering scality-glance-store-tox » pep8,ubuntu14-largeConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: All nodes of label ‘centos6-large’ are offlineConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: centos6-large-589 is offlineConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: Waiting for next available executor on centos6-large-589scality-glance-store-tox » py26,centos6-large completed with result FAILUREscality-glance-store-tox » py27,centos7-large completed with result FAILUREscality-glance-store-tox » py27,ubuntu14-large completed with result FAILUREscality-glance-store-tox » py27,ubuntu12-large completed with result FAILUREscality-glance-store-tox » pep8,ubuntu14-large completed with result UNSTABLESetting status of 71d78e08c89284682819ba0bada03ddf05bd0ab8 to FAILURE with url https://37.187.159.67:5443/job/scality-glance-store-tox/19/ and message: Merged build finished.
Test FAILed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-functional-tests/27/
Test PASSed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-functional-tests/28/<h2>Build result: FAILURE</span></h2>[...truncated 9 lines...] > git config remote.origin.url https://github.com/scality/scality-glance-store.git # timeout=10 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10 > git config remote.origin.url https://github.com/scality/scality-glance-store.git # timeout=10Fetching upstream changes from https://github.com/scality/scality-glance-store.git > git -c core.askpass=true fetch --tags --progress https://github.com/scality/scality-glance-store.git +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/* > git rev-parse refs/remotes/origin/pr/8/merge^{commit} # timeout=10 > git rev-parse refs/remotes/origin/origin/pr/8/merge^{commit} # timeout=10Checking out Revision ca4b1fec2bda5c9f1069be7a11a9553f354c3f0b (refs/remotes/origin/pr/8/merge) > git config core.sparsecheckout # timeout=10 > git checkout -f ca4b1fec2bda5c9f1069be7a11a9553f354c3f0b > git rev-list cd60e534cc8c3bea9faa7d94605ab9b6fc946701 # timeout=10First time build. Skipping changelog. > git remote # timeout=10 > git submodule init # timeout=10 > git submodule sync # timeout=10 > git config --get remote.origin.url # timeout=10 > git submodule update --init --recursiveTriggering scality-glance-store-functional-tests » ubuntu14-largeConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: Waiting for next available executor on ubuntu14-large-5a7Configuration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: Waiting for next available executor on ubuntu14-largeConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: All nodes of label ‘ubuntu14-large’ are offlineConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: Waiting for next available executor on ubuntu14-largescality-glance-store-functional-tests » ubuntu14-large completed with result FAILURESetting status of 71d78e08c89284682819ba0bada03ddf05bd0ab8 to FAILURE with url https://37.187.159.67:5443/job/scality-glance-store-functional-tests/28/ and message: Merged build finished.
Test FAILed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-functional-tests/29/<h2>Build result: FAILURE</span></h2>[...truncated 5 lines...] > git config remote.origin.url https://github.com/scality/scality-glance-store.git # timeout=10Fetching upstream changes from https://github.com/scality/scality-glance-store.git > git --version # timeout=10 > git -c core.askpass=true fetch --tags --progress https://github.com/scality/scality-glance-store.git +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/* > git rev-parse refs/remotes/origin/pr/8/merge^{commit} # timeout=10 > git rev-parse refs/remotes/origin/origin/pr/8/merge^{commit} # timeout=10Checking out Revision 252102c6efca11be0b72ed4ea6b0bf9bef7e126f (refs/remotes/origin/pr/8/merge) > git config core.sparsecheckout # timeout=10 > git checkout -f 252102c6efca11be0b72ed4ea6b0bf9bef7e126f > git rev-list ca4b1fec2bda5c9f1069be7a11a9553f354c3f0b # timeout=10First time build. Skipping changelog. > git remote # timeout=10 > git submodule init # timeout=10 > git submodule sync # timeout=10 > git config --get remote.origin.url # timeout=10 > git submodule update --init --recursiveTriggering scality-glance-store-functional-tests » ubuntu14-largeConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: All nodes of label ‘ubuntu14-large’ are offlineConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: ubuntu14-large-c2d is offlineConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: Waiting for next available executorConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: ubuntu14-large-b2c is offlineConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: Waiting for next available executor on ubuntu14-large-b2cscality-glance-store-functional-tests » ubuntu14-large completed with result FAILURESetting status of e69b70f26add897b80ff3b148220ec7344ae7e97 to FAILURE with url https://37.187.159.67:5443/job/scality-glance-store-functional-tests/29/ and message: Merged build finished.
Test FAILed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/20/<h2>Build result: ABORTED</span></h2>[...truncated 8 lines...] > git -c core.askpass=true fetch --tags --progress https://github.com/scality/scality-glance-store.git +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/* > git rev-parse refs/remotes/origin/pr/8/merge^{commit} # timeout=10 > git rev-parse refs/remotes/origin/origin/pr/8/merge^{commit} # timeout=10Checking out Revision 252102c6efca11be0b72ed4ea6b0bf9bef7e126f (refs/remotes/origin/pr/8/merge) > git config core.sparsecheckout # timeout=10 > git checkout -f 252102c6efca11be0b72ed4ea6b0bf9bef7e126fFirst time build. Skipping changelog.Triggering scality-glance-store-tox » py26,centos6-largeTriggering scality-glance-store-tox » py27,centos7-largeTriggering scality-glance-store-tox » py27,ubuntu14-largeTriggering scality-glance-store-tox » py27,ubuntu12-largeTriggering scality-glance-store-tox » pep8,ubuntu14-largeConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: All nodes of label ‘centos6-large’ are offlineConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: Waiting for next available executorConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: All nodes of label ‘centos6-large’ are offlineConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: centos6-large-6ad is offlineConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: Waiting for next available executor on centos6-large-6adscality-glance-store-tox » py26,centos6-large completed with result FAILUREscality-glance-store-tox » py27,centos7-large completed with result FAILUREAbortedAborted by Jordan PittierInterrupting #20Interrupting #20Setting status of e69b70f26add897b80ff3b148220ec7344ae7e97 to FAILURE with url https://37.187.159.67:5443/job/scality-glance-store-tox/20/ and message: Merged build finished.
Test FAILed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-functional-tests/30/<h2>Build result: FAILURE</span></h2>[...truncated 2 lines...]Building on master in workspace /var/lib/jenkins/workspace/scality-glance-store-functional-tests > git rev-parse --is-inside-work-tree # timeout=10Fetching changes from the remote Git repository > git config remote.origin.url https://github.com/scality/scality-glance-store.git # timeout=10Fetching upstream changes from https://github.com/scality/scality-glance-store.git > git --version # timeout=10 > git -c core.askpass=true fetch --tags --progress https://github.com/scality/scality-glance-store.git +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/* > git rev-parse refs/remotes/origin/pr/8/merge^{commit} # timeout=10 > git rev-parse refs/remotes/origin/origin/pr/8/merge^{commit} # timeout=10Checking out Revision 2d7cba2fd95e9100c707b513cc36823dbe946f10 (refs/remotes/origin/pr/8/merge) > git config core.sparsecheckout # timeout=10 > git checkout -f 2d7cba2fd95e9100c707b513cc36823dbe946f10 > git rev-list 252102c6efca11be0b72ed4ea6b0bf9bef7e126f # timeout=10 > git remote # timeout=10 > git submodule init # timeout=10 > git submodule sync # timeout=10 > git config --get remote.origin.url # timeout=10 > git submodule update --init --recursiveTriggering scality-glance-store-functional-tests » ubuntu14-largeConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: All nodes of label ‘ubuntu14-large’ are offlineConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: ubuntu14-large-678 is offlineConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: Waiting for next available executor on ubuntu14-large-678scality-glance-store-functional-tests » ubuntu14-large completed with result FAILURESetting status of f239b690b8ae374084fbbc4ad012988b44ca214d to FAILURE with url https://37.187.159.67:5443/job/scality-glance-store-functional-tests/30/ and message: Merged build finished.
Test FAILed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/21/
Test PASSed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-functional-tests/31/<h2>Build result: FAILURE</span></h2>[...truncated 3 lines...] > git rev-parse --is-inside-work-tree # timeout=10Fetching changes from the remote Git repository > git config remote.origin.url https://github.com/scality/scality-glance-store.git # timeout=10Fetching upstream changes from https://github.com/scality/scality-glance-store.git > git --version # timeout=10 > git -c core.askpass=true fetch --tags --progress https://github.com/scality/scality-glance-store.git +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/* > git rev-parse refs/remotes/origin/pr/8/merge^{commit} # timeout=10 > git rev-parse refs/remotes/origin/origin/pr/8/merge^{commit} # timeout=10Checking out Revision d095910f9a9dea3e21b6aae47c4ab8c0b9ec6fdb (refs/remotes/origin/pr/8/merge) > git config core.sparsecheckout # timeout=10 > git checkout -f d095910f9a9dea3e21b6aae47c4ab8c0b9ec6fdb > git rev-list 2d7cba2fd95e9100c707b513cc36823dbe946f10 # timeout=10 > git remote # timeout=10 > git submodule init # timeout=10 > git submodule sync # timeout=10 > git config --get remote.origin.url # timeout=10 > git submodule update --init --recursiveTriggering scality-glance-store-functional-tests » ubuntu14-largeConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: All nodes of label ‘ubuntu14-large’ are offlineConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: ubuntu14-large-1eb is offlineConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: All nodes of label ‘ubuntu14-large’ are offlineConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: Waiting for next available executor on ubuntu14-largescality-glance-store-functional-tests » ubuntu14-large completed with result FAILURESetting status of 5d2f0e750d2fee44eb58776909b2a9459c91e6ba to FAILURE with url https://37.187.159.67:5443/job/scality-glance-store-functional-tests/31/ and message: Merged build finished.
Test FAILed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/22/<h2>Build result: FAILURE</span></h2>[...truncated 5 lines...] > git config remote.origin.url https://github.com/scality/scality-glance-store.git # timeout=10Fetching upstream changes from https://github.com/scality/scality-glance-store.git > git --version # timeout=10 > git -c core.askpass=true fetch --tags --progress https://github.com/scality/scality-glance-store.git +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/* > git rev-parse refs/remotes/origin/pr/8/merge^{commit} # timeout=10 > git rev-parse refs/remotes/origin/origin/pr/8/merge^{commit} # timeout=10Checking out Revision d095910f9a9dea3e21b6aae47c4ab8c0b9ec6fdb (refs/remotes/origin/pr/8/merge) > git config core.sparsecheckout # timeout=10 > git checkout -f d095910f9a9dea3e21b6aae47c4ab8c0b9ec6fdbFirst time build. Skipping changelog.Triggering scality-glance-store-tox » py26,centos6-largeTriggering scality-glance-store-tox » py27,centos7-largeTriggering scality-glance-store-tox » py27,ubuntu14-largeTriggering scality-glance-store-tox » py27,ubuntu12-largeTriggering scality-glance-store-tox » pep8,ubuntu14-largeConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: All nodes of label ‘centos6-large’ are offlineConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: centos6-large-cc is offlineConfiguration scality-glance-store-tox » py26,centos6-large is still in the queue: Waiting for next available executor on centos6-large-ccscality-glance-store-tox » py26,centos6-large completed with result SUCCESSscality-glance-store-tox » py27,centos7-large completed with result SUCCESSscality-glance-store-tox » py27,ubuntu14-large completed with result SUCCESSscality-glance-store-tox » py27,ubuntu12-large completed with result FAILUREscality-glance-store-tox » pep8,ubuntu14-large completed with result SUCCESSSetting status of 5d2f0e750d2fee44eb58776909b2a9459c91e6ba to FAILURE with url https://37.187.159.67:5443/job/scality-glance-store-tox/22/ and message: Merged build finished.
Test FAILed.

Commit:
Use easy_install to install pip

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-functional-tests/32/<h2>Build result: FAILURE</span></h2>[...truncated 3 lines...] > git rev-parse --is-inside-work-tree # timeout=10Fetching changes from the remote Git repository > git config remote.origin.url https://github.com/scality/scality-glance-store.git # timeout=10Fetching upstream changes from https://github.com/scality/scality-glance-store.git > git --version # timeout=10 > git -c core.askpass=true fetch --tags --progress https://github.com/scality/scality-glance-store.git +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/* > git rev-parse refs/remotes/origin/pr/8/merge^{commit} # timeout=10 > git rev-parse refs/remotes/origin/origin/pr/8/merge^{commit} # timeout=10Checking out Revision 175e17770dca066486d5191eef6c764d4cceb62b (refs/remotes/origin/pr/8/merge) > git config core.sparsecheckout # timeout=10 > git checkout -f 175e17770dca066486d5191eef6c764d4cceb62b > git rev-list d095910f9a9dea3e21b6aae47c4ab8c0b9ec6fdb # timeout=10 > git remote # timeout=10 > git submodule init # timeout=10 > git submodule sync # timeout=10 > git config --get remote.origin.url # timeout=10 > git submodule update --init --recursiveTriggering scality-glance-store-functional-tests » ubuntu14-largeConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: All nodes of label ‘ubuntu14-large’ are offlineConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: ubuntu14-large-859 is offlineConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: All nodes of label ‘ubuntu14-large’ are offlineConfiguration scality-glance-store-functional-tests » ubuntu14-large is still in the queue: Waiting for next available executor on ubuntu14-largescality-glance-store-functional-tests » ubuntu14-large completed with result FAILURESetting status of b1e5455dd889a6ae20f09a449d4aa5b989f216b4 to FAILURE with url https://37.187.159.67:5443/job/scality-glance-store-functional-tests/32/ and message: Merged build finished.
Test FAILed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/23/
Test PASSed.

Commit:
Devstack: use FORCE=yes for centos 6 compatibility

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-functional-tests/36/<h2>Build result: FAILURE</span></h2>[...truncated 6 lines...]Fetching upstream changes from https://github.com/scality/scality-glance-store.git > git --version # timeout=10 > git -c core.askpass=true fetch --tags --progress https://github.com/scality/scality-glance-store.git +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/* > git rev-parse refs/remotes/origin/pr/8/merge^{commit} # timeout=10 > git rev-parse refs/remotes/origin/origin/pr/8/merge^{commit} # timeout=10Checking out Revision 9a38c98aaa9a47a00597f04bbc5375c0c7887e43 (refs/remotes/origin/pr/8/merge) > git config core.sparsecheckout # timeout=10 > git checkout -f 9a38c98aaa9a47a00597f04bbc5375c0c7887e43 > git rev-list 175e17770dca066486d5191eef6c764d4cceb62b # timeout=10 > git remote # timeout=10 > git submodule init # timeout=10 > git submodule sync # timeout=10 > git config --get remote.origin.url # timeout=10 > git submodule update --init --recursiveTriggering scality-glance-store-functional-tests » centos6-largeTriggering scality-glance-store-functional-tests » ubuntu14-largeTriggering scality-glance-store-functional-tests » ubuntu12-largeConfiguration scality-glance-store-functional-tests » centos6-large is still in the queue: All nodes of label ‘centos6-large’ are offlineConfiguration scality-glance-store-functional-tests » centos6-large is still in the queue: centos6-large-6e0 is offlineConfiguration scality-glance-store-functional-tests » centos6-large is still in the queue: Waiting for next available executor on centos6-large-6e0scality-glance-store-functional-tests » centos6-large completed with result FAILUREscality-glance-store-functional-tests » ubuntu14-large completed with result FAILUREscality-glance-store-functional-tests » ubuntu12-large completed with result FAILURESetting status of 93e01838e1df04a1820efe314765186c5da6e3f9 to FAILURE with url https://37.187.159.67:5443/job/scality-glance-store-functional-tests/36/ and message: Merged build finished.
Test FAILed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/24/
Test PASSed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/25/
Test PASSed.

scality commented:
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/26/
Test PASSed.

NicolasT commented:
LGTM
